tests/reports/unit/test_sonatadata.cpp

```diff
@@ -120,16 +114,16 @@ SCENARIO("Test SonataData class", "[SonataData][IOWriter]") {
             THEN("We check that the spike nodes ids are ordered according to timestamps") {
                 const std::vector<int> node_ids = sonata_spikes->get_spike_node_ids();
                 std::vector<int> compare = {5, 2, 3, 2, 3};
-                if (global_size == 1) {
+                #ifdef HAVE_MPI
                     REQUIRE(node_ids == compare);
-                }
+                #endif
```

Comment:
Why is this only with MPI? Above it was unconditional…

Reply:
I had this because there was no sort spikes in serial implementation, however this is solved in the latest commit.